### DISCUSSION TITLE
Where to configure AdGuard Home for translating IP Address ?

### DISCUSSION THREAD

**Discussion by titithebad** [AdguardTeam/AdGuardHome#6602] [Q&A]
Dear all,

I installed AdGuard Home on my private network with a lot of servers. Where is it possible to concfigure AdGuard for translating IP Address such as 192.168.x.y into a name suchase toto.tata ?

Regards !

**Comment by ovargasp** [comment]
You need to create a "Custom filtering rule" the rule has to have this syntax: x.x.x.x domain, in your example is: 192.168.x.y toto.tata

you can read about it here: https://github.com/AdguardTeam/AdGuardHome/wiki/Hosts-Blocklists#etc-hosts
